src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.navigation__bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: rgb(255 255 255);
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav_item {
+  position: relative;
+  padding: 0;
+  margin-left: 20px;
+  font-size: 12px;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+  position: absolute;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_link {
+  color: black;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
You are using this color 3 times, so it would be good to use here css variables to make color modification easier.
Create variable for the color :)